app/controllers/books_controller.rb

```diff
@@ -0,0 +1,16 @@
+class BooksController < ApplicationController
+  include Wor::Paginate
+  before_action :book, only: [:show]
```

Comment:
We don't usually use `before_action`s a lot because it's hard to follow what code is actually being executed, but it would be correct if we had many methods that need this. Since in this case it's only one method, I'd move the line to the actual method and remove this.

But also, if you change `render json: @book` for `render json: book` it should accomplish what you need even easier because it'll call the method directly.